src/components/Chip/Chip.styles.tsx

```diff
@@ -3,7 +3,7 @@ import styled, { css } from 'styled-components';
 import { yellow50, yellow60, brown100 } from '../../core/GlobalStyle/variable';
 import { font, textEllipsis } from '../../core/GlobalStyle/mixin';
 
-export const Container = styled.div`
+export const Container = styled.div<{ isOverflowed: boolean }>`
```

Comment:
Chip 컴포넌트의 isOverflowed를 optional로 함에 따라 아래의 방법중 하나로 같이 변경해 주면 좋을 것 같슴다!

```suggestion
export const Container = styled.div<{ isOverflowed: boolean = false }>`
```

```suggestion
export const Container = styled.div<{ isOverflowed?: boolean }>`
```